unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -92,6 +92,34 @@
         are not affected.
       </description>
     </request>
+
+    <event name="ping" since="3">
+      <description summary="check if the client is alive">
+        The ping event asks the client if it's still alive. Clients must
+        make a pong request passing the provided serial back to the compositor.
+
+        It's unspecified what will happen if the client doesn't respond to
+        the ping request, or in what time frame.
+
+        The compositor may use this request in order to achieve frame-perfect
+        handling of new outputs. Because of this, if a client wishes to
+        create a new layer surface in response to the creation of a new
+        output global, the client should make the get_layer_surface request
+        immediately on receiving the wl_registry.global event with the new
+        output and before processing any further events.
```

Comment:
Hm. These is one issue with this approach though: if the client needs to preform more requests to decide whether to create a new layer-shell surface or not. A classic example of this is a client that only creates a layer-shell surface on an output named "DP-1": the client would need to retrieve the output name via xdg-output prior to creating the layer surface.

Reply:
Yeah, that's a problem. I think we may need to go with the special purpose `new_output`/`ack_new_output` version because of this. However a ping/pong may also be worth adding for other use cases (and without this special language).